### ISSUE TITLE
Background is shown in the padding even if padding modifier is applied after the background modifier

### ISSUE THREAD

**Issue by ivteplo** [teplovs/bon-ui#7]
Background has to be shown in padding when padding modifier was applied before background modifier.

Example when background has to be shown in padding:
```javascript
class MyView extends View {
    body () {
        return (
            new Text("Hello world!")
                .padding()
                .background(Color.gray)
        )
    }
}
```

Example when background has to __not__ be shown in padding:
```javascript
class MyView extends View {
    body () {
        return (
            new Text("Hello world!")
                .background(Color.gray)
                .padding()
        )
    }
}
```

But background is shown in padding when running the code. This needs to be fixed

**Comment by ivteplo** [comment]
Have to add extra wrapper (`ContainerVNode` for example) for `background` that inherits padding and has absolute positioning.

Structure:
```pug
div.zstack
    div.background-wrapper (style='position: absolute; top: 0; left: 0; width: 100%; height: 100%; padding: inherit')
        div.background (style='width: 100%; height: 100%')
    div.child
```

**Comment by ivteplo** [comment]
Fixed in [`75cead1`](https://github.com/teplovs/bon-ui/commit/75cead12b25f5559a15290fabc3b8226ece72b93)
